### PR TITLE
Add README

### PR DESCRIPTION
**Pull request by lantiga**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,137 @@
...
+
+Hackable implementation of the [StableLM](https://github.com/Stability-AI/StableLM) and [Pythia](https://github.com/EleutherAI/pythia) family of models released under the **Apache 2.0 license.**
+
+This implementation builds on [Lit-LLaMA](https://github.com/lightning-AI/lit-llama) and [nanoGPT](https://github.com/karpathy/nanoGPT), and it's powered by [Lightning Fabric](https://lightning.ai/docs/fabric/stable/) ⚡.
+
+Weights are available under the Apache 2.0 license and can be downloaded [following these instructions](howto/download_weights.md).
+
+## Design principles
+
+This repository follows the **Lit design principle** of **openness through clarity**.
```
Comment (carmocca) [inline README.md:30]: As a new reader, it's not clear to me that a "lit design principle" is a properly defined term

Maybe this?

```suggestion
This repository follows the **design principle** of **openness through clarity**.
```

File: README.md [hunk 2]
```diff
@@ -1 +1,137 @@
...
+
+&nbsp;
+
+## Setup
+
+Clone the repo
+
+```bash
+git clone https://github.com/Lightning-AI/lit-stablelm
+cd lit-llama
```
Comment (carmocca) [inline README.md:52]:
```suggestion
cd lit-stablelm
```

File: README.md [hunk 3]
```diff
@@ -1 +1,137 @@
...
+
+To generate text predictions, you need to download the model weights. **If you don't have them, check out our [guide](howto/download_weights.md).**
+
+Run inference:
+
+```bash
+python generate.py --prompt "Hello, my name is"
+```
+
+This will run the 7B model and require ~26 GB of GPU memory (A100 GPU).
```
Comment (carmocca) [inline README.md:75]: It's currently configured to run the 3b pre-trained model. We need to update these numbers. Can be done in a follow-up

File: README.md [hunk 4]
```diff
@@ -1 +1,129 @@
...
+We are on a quest towards fully open source AI.
+
+<img align="right" src="https://pl-public-data.s3.amazonaws.com/assets_lightning/Lit_LLaMA_Illustration3x.png" alt="Lit-LLaMA" width="128"/>
+
+Join us and start contributing, especially on the following areas:
+
+- [ ] [Pre-training](https://github.com/Lightning-AI/lit-llama/labels/pre-training)
+- [ ] [Fine-tuning (full and LoRA)](https://github.com/Lightning-AI/lit-llama/labels/fine-tuning)
+- [ ] [Quantization](https://github.com/Lightning-AI/lit-llama/labels/quantization)
+- [ ] [Sparsification](https://github.com/Lightning-AI/lit-llama/labels/sparsification)
```
Comment (carmocca) [inline README.md:111]: Is this list still worth having? Most have been addressed already

File: README.md [hunk 5]
```diff
@@ -1 +1,129 @@
...
+
+We welcome all individual contributors, regardless of their level of experience or hardware. Your contributions are valuable, and we are excited to see what you can accomplish in this collaborative and supportive environment. 
+
+Unsure about contributing? Check out our [Contributing to Lit-LLaMA: A Hitchhiker’s Guide to the Quest for Fully Open-Source AI](https://lightning.ai/pages/community/tutorial/contributing-to-lit-llama-a-hitchhikers-guide-to-the-quest-for-fully-open-source-ai/) guide.
```
Comment (carmocca) [inline README.md:115]:
```suggestion
Unsure about contributing? Check out our [Contributing to Lit-LLaMA: A Hitchhiker’s Guide to the Quest for Fully Open-Source AI](https://lightning.ai/pages/community/tutorial/contributing-to-lit-llama-a-hitchhikers-guide-to-the-quest-for-fully-open-source-ai/) guide. The same guidelines apply to Lit-StableLM
```

**Review by carmocca — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (lantiga) [reply README.md]: Fixed it

File: README.md (hunk 4 above)
Comment (lantiga) [reply README.md]: True but it's good call to action (there's always more to)
